application/MainWindowRB.cpp

```diff
@@ -59,39 +60,43 @@ MainWindowRB::MainWindowRB(void)
     ;
 
     ImageButton* fullscreenButton = new ImageButton("fullscreen",
-        new BMessage(MSG_FULLSCREEN) , 0.3, 1);
+        new BMessage(MSG_FULLSCREEN) , 0.3, 1, "Fullscreen");
     fullscreenButton->SetExplicitMinSize(BSize(fMaxHeight, 0));
     fullscreenButton->SetExplicitMaxSize(BSize(fMaxHeight, fMaxHeight));
+
     ImageButton* fitPageHeight = new ImageButton("fit_page_height",
-        new BMessage(MSG_FIT_PAGE_HEIGHT), 0.3, 1);
+        new BMessage(MSG_FIT_PAGE_HEIGHT), 0.3, 1, "Fit document vertically");
     fitPageHeight->SetExplicitMinSize(BSize(fMaxHeight, fMaxHeight));
     fitPageHeight->SetExplicitMaxSize(BSize(fMaxHeight, fMaxHeight));
+
     ImageButton* fitPageWidth = new ImageButton("fit_page_width",
-        new BMessage(MSG_FIT_PAGE_WIDTH), 0.3, 1);
+        new BMessage(MSG_FIT_PAGE_WIDTH), 0.3, 1, "Fit document horizontally");
     fitPageWidth->SetExplicitMinSize(BSize(fMaxHeight, fMaxHeight));
     fitPageWidth->SetExplicitMaxSize(BSize(fMaxHeight, fMaxHeight));
 
-
-	ImageButton* zoomOriginal = new ImageButton("zoom_original", new BMessage(MSG_NO_ZOOM), 0.3, 1);
+	ImageButton* zoomOriginal = new ImageButton("zoom_original", 
+		new BMessage(MSG_NO_ZOOM), 0.3, 1, "Reset zoom");
     zoomOriginal->SetExplicitMinSize(BSize(fMaxHeight, fMaxHeight));
     zoomOriginal->SetExplicitMaxSize(BSize(fMaxHeight, fMaxHeight));
-    
+
     ImageButton* zoomOut = new ImageButton("zoom_out",
-        new BMessage(MSG_ZOOM_OUT), 0.3, 1);
+        new BMessage(MSG_ZOOM_OUT), 0.3, 1, "Zoom out");
     zoomOut->SetExplicitMinSize(BSize(fMaxHeight, fMaxHeight));
     zoomOut->SetExplicitMaxSize(BSize(fMaxHeight, fMaxHeight));
+
     ImageButton* zoomIn = new ImageButton("zoom_in",
-        new BMessage(MSG_ZOOM_IN), 0.3, 1);
+        new BMessage(MSG_ZOOM_IN), 0.3, 1, "Zoom in");
     zoomIn->SetExplicitMinSize(BSize(fMaxHeight, fMaxHeight));
     zoomIn->SetExplicitMaxSize(BSize(fMaxHeight, fMaxHeight));
-    ImageButton* rotateLeftButton = new ImageButton("rotate_left", nullptr, 0.3, 1);
+
+    ImageButton* rotateLeftButton = new ImageButton("rotate_left", nullptr, 0.3, 1, "Rotate left");
     rotateLeftButton->SetExplicitMinSize(BSize(fMaxHeight, fMaxHeight));
     rotateLeftButton->SetExplicitMaxSize(BSize(fMaxHeight, fMaxHeight));
-    ImageButton* rotateRightButton = new ImageButton("rotate_right", nullptr, 0.3, 1);
+
+    ImageButton* rotateRightButton = new ImageButton("rotate_right", nullptr, 0.3, 1, "Rotate right");
```

Comment:
See the above.

Reply:
They got reversed? Looks alright to me...

Reply:
Weird. Now they look correct. Oh well.

Reply:
Please break this line.